test/unit/cli.test.js

```diff
@@ -227,6 +227,7 @@ describe('cli parser', function () {
             '--export-environment exported_env.json ' +
             '--export-globals exported_glob.json ' +
             '--reporter-cli-no-summary ' +
+            '--reporter-cli-no-success-assertions ' +
```

Comment:
This will also have to be tested (see the sequence of `expect` statements below) 😄

Reply:
This should be done now. Can you check it?